src/api/orders/orders.request.ts

```diff
@@ -21,3 +21,21 @@ export class ActiveOrdersRequest {
         }
     }
 }
+
+export class OrderStatusRequest {
+    public readonly orderHash: string
+
+    constructor(params: OrderStatusParams) {
+        this.orderHash = params.orderHash
+    }
+
+    static new(params: OrderStatusParams): OrderStatusRequest {
+        return new OrderStatusRequest(params)
+    }
+
+    build(): OrderStatusParams {
+        return {
+            orderHash: this.orderHash
+        }
+    }
+}
```

Comment:
Add `validation` function like here:
https://github.com/1inch/fusion-sdk/blob/20679a15cea5b49bd0f86b5fb237726d80d09b01/src/api/quoter/quoter.request.ts#L32

Check that `orderHash.length === 66` and the string is hexdecimal

Reply:
done